migrations/versions/23e65ab635fa_.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
+
+Revision ID: 23e65ab635fa
+Revises: f748d763fc8
+Create Date: 2015-03-29 11:20:38.824318
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '23e65ab635fa'
+down_revision = 'f748d763fc8'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('item', sa.Column('trending', sa.Integer(), nullable=True))
```

Comment:
Do we need a whole column for this? Can't we count total votes from our UserItem table pretty quickly?